intellij/src/saros/intellij/ui/tree/SessionAndContactsTreeView.java

```diff
@@ -113,6 +126,7 @@ public SessionAndContactsTreeView(@NotNull Project project) {
   @Override
   public void dispose() {
     connectionHandler.removeConnectionStateListener(connectionStateListener);
+    xmppContactsService.removeListener(contactsUpdateListener);
```

Comment:
Isnt this widget missing some kind of isDisposed() handling ? The events could be triggered while a dispose is in progress,